process/tests/test_process.py

```diff
@@ -91,45 +91,45 @@ def test_psutil_wrapper_accessors(aggregator):
 
 def test_psutil_wrapper_accessors_fail(aggregator):
     # Load check with empty config
-    process = ProcessCheck(common.CHECK_NAME, {}, {})
+    process = ProcessCheck(common.CHECK_NAME, {}, [{}])
     meminfo = process.psutil_wrapper(get_psutil_proc(), 'memory_infoo', ['rss', 'vms'], False)
     assert 'rss' not in meminfo
     assert 'vms' not in meminfo
 
 
 @patch('psutil.process_iter', side_effect=[[NamedMockProcess("Process 1")], [NamedMockProcess("Process 2")]])
-def test_process_list_cache(aggregator):
+def test_process_list_cache(aggregator, dd_run_check):
     config = {
         'instances': [{'name': 'python', 'search_string': ['python']}, {'name': 'python', 'search_string': ['python']}]
     }
     process1 = ProcessCheck(common.CHECK_NAME, {}, [config['instances'][0]])
     process2 = ProcessCheck(common.CHECK_NAME, {}, [config['instances'][1]])
 
     with patch('datadog_checks.process.cache.ProcessListCache.reset'):
-        process1.check(config['instances'][0])
-        process2.check(config['instances'][1])
+        dd_run_check(process1)
+        dd_run_check(process2)
 
     # Should always succeed
     assert process1.process_list_cache.elements[0].name() == "Process 1"
     # Fails with 'assert "Process 2" == "Process 1"' if process list cache is not shared
     assert process2.process_list_cache.elements[0].name() == "Process 1"
 
 
-def test_ad_cache(aggregator):
+def test_ad_cache(aggregator, dd_run_check):
     config = {'instances': [{'name': 'python', 'search_string': ['python'], 'ignore_denied_access': 'false'}]}
     process = ProcessCheck(common.CHECK_NAME, {}, config['instances'])
 
     def deny_name(obj):
         raise psutil.AccessDenied()
 
     with patch.object(psutil.Process, 'name', deny_name):
-        with pytest.raises(psutil.AccessDenied):
-            process.check(config['instances'][0])
+        with pytest.raises(Exception):
```

Comment:
Why was this changed?

Reply:
https://github.com/DataDog/integrations-core/blob/4c7725c9f1be4985381aad9740e7186f16a87976/datadog_checks_dev/datadog_checks/dev/plugin/pytest.py#L211-L218

Reply:
Thanks, makes sense!